jaeger-apachehttpclient/src/main/java/com/uber/jaeger/httpclient/TracingInterceptors.java

```diff
@@ -1,23 +1,15 @@
 /*
  * Copyright (c) 2016, Uber Technologies, Inc
```

Comment:
Shouldn't this be 2017? We can't apply the license retroactively, can we?

Reply:
Changing the licensing terms does not change the copyright. There are only two files I corrected in this PR that had incorrect 2017 copyright, even though the files were written in 2016, so I changed them to 2016-2017.